conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: numpy
+  version: 1.11.0{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}
```

Comment:
The main reason I put it in the top level directory as I am trying to follow the canonical example. ( https://github.com/conda/conda-build ) Normally, it should be in the top level directory like `setup.py` so that it is visible and easy to use in that same manner.

The purpose of this, at present, is just to provide an easy way for people to build NumPy from development. It doesn't upload any packages. Though that is certainly something we could do and should discuss.